product/charts/miq_reports/vim_perf_daily_middleware_datasource.yaml

```diff
@@ -0,0 +1,88 @@
+#
+# This is an MIQ Report configuration file
+#   Single value parameters are specified as:
+#     single_value_parm: value
+#   Multiple value parameters are specified as:
+#     multi_value_parm:
+#       - value 1
+#       - value 2
+#
+
+# Report title
+title: "VIM Performance Daily"

```

Comment:
Some legacy definition of the Virtual Models used for the reports used on the graphs.
Those are under product/charts/miq_reports and all files used that convention for naming and also inside the reports.
Probably it can be different but I wanted to follow same convention and to not change the current namespace used.